### PR TITLE
Travis ci

### PR DESCRIPTION
**Pull request by caa06d9c**

### CODE REVIEW

File: docker/build-debian.sh [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+ 'scons' \
+)
+
+TEST_TOOLS=(
+ 'bc' \
+ 'nano' \
+ 'netcat' \
+ 'valgrind' \
+)
+
```
Comment (kzangeli) [inline docker/build-debian.sh:70]: For centos you have mongo as part of the test tools. Not necessary for debian?

Comment (caa06d9c) [reply docker/build-debian.sh:70]: Debian uses another repository, there is a special block for it

Comment (kzangeli) [reply docker/build-debian.sh:70]: ok, great. Just checking ... :)

**Comment by kzangeli** [comment]
LGTM

**Comment by caa06d9c** [comment]
IPv6 still not working, so this PR should not be merged

**Comment by kzangeli** [comment]
LGTM (some of the failing tests need to be looked into, can do that later on)

**Pull request merged**